permission_handler/example/lib/plugin_example/permission_widget.dart

```diff
@@ -17,7 +17,7 @@ class _PermissionState extends State<PermissionWidget> {
   _PermissionState(this._permission);
 
   final Permission _permission;
-  PermissionStatus _permissionStatus = PermissionStatus.undetermined;
+  PermissionStatus? _permissionStatus = PermissionStatus.undetermined;
```

Comment:
Permission status should never be `null` (see also other remarks).
```suggestion
  PermissionStatus _permissionStatus = PermissionStatus.undetermined;
```